pages/telecom/sms/smpp-specification/guide.en-gb.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: SMPP Technical Specifications
 excerpt: 'Discover the technical specifications of the OVHcloud SMPP solution'
-updated: 2023-07-25
+updated: 2023-07-31
```

Comment:
```suggestion
updated: 2023-08-18
```

Reply:
Would be great if a GH action can do this automatically :)